api/transaction/routes.go

```diff
@@ -180,3 +182,41 @@ func txResponseFromTransaction(tx *transaction.Transaction) TxResponse {
 
 	return response
 }
+
+// GetTransactionCost returns how many gas a transaction wil consume
+func GetTransactionCost(c *gin.Context) {
```

Comment:
rename this to `ComputeTransactionCostHandler` or something similar as this may lead to confusion because this is not a handler for a `GET` endpont

Reply:
done.